src/test/scala/temple/builder/ServerBuilderTest.scala

```diff
@@ -62,54 +61,13 @@ class ServerBuilderTest extends FlatSpec with Matchers {
     )
   }
 
-  it should "build a correct simple ServiceRoot with no endpoints" in {
-    val serviceRoot: ServiceRoot = BuilderTestData.simpleTemplefile.allServicesWithPorts.head match {
-      case (name, service, port) =>
-        ServerBuilder
```

Comment:
Why has this test gone?

Reply:
Surely we should still be testing a `Templefile` where all endpoints are omitted?